src/commercial.consentless.ts

```diff
@@ -20,15 +23,26 @@ const bootConsentless = async (consentState: ConsentState): Promise<void> => {
  	*/
 	maybeUnsetAdFreeCookie(AdFreeCookieReasons.ConsentOptOut);
 
-	await Promise.all([
+	const consentlessModuleList = [
 		setAdTestCookie(),
 		setAdTestInLabelsCookie(),
 		initConsentless(consentState),
 		initExclusionSlot(),
 		initFixedSlots(),
 		initArticleInline(),
 		initLiveblogInline(),
-	]);
+	];
+
+	if (isDotcomRendering) {
+		const userFeatures = await import(
```

Comment:
This doesn't require any action, just noting down something I noticed:

It looks like Webpack doesn't create a new chunk for lib/user-features for this dynamic import, which is what I would've expected. It seems instead it just includes it in the main standalone/consentless bundles, I guess because we're already imported it statically elsewhere?

![Screenshot 2023-07-24 at 13 17 47](https://github.com/guardian/commercial/assets/8000415/51f52fc0-149b-44d7-ad62-8b1425da22f0)